samples/WebAPI/Controllers/OrderProcessingController.cs

```diff
@@ -31,8 +32,7 @@ public async Task<ActionResult> CreateOrder([FromBody] OrderInfo orderInfo)
         // Generate an order ID and start the order processing workflow orchestration
         string orderId = $"{orderInfo.Item}-{Guid.NewGuid().ToString()[..4]}";
         await this.durableTaskClient.ScheduleNewProcessOrderOrchestratorInstanceAsync(
-            instanceId: orderId,
-            input: orderInfo);
+            orderInfo, new() { InstanceId = orderId });
```

Comment:
Do I recall correctly that you had a `SubOrchestratorOptions.WithInstanceId(...)` or similar method? If so, that might be a nicer snippet to use here to make the code a bit more readable, and to make it more obvious that there are other options that can be configured. I like the `new()` pattern, but only when the type it refers to is obvious, which isn't the case here.

Reply:
That is for sub orchestrations, this one would be `StartOchestrationOptions`, which does not inherit from the same `TaskOptions`.

But yes, I can make the type explicit.